src/api/c++/z3++.h

```diff
@@ -274,7 +301,7 @@ namespace z3 {
         /**
            \brief Return a floating point sort.
            \c ebits is a number of exponent bits,
-           \c sbits	is a number of significand bits,
+           \c sbits    is a number of significand bits,
```

Comment:
random change

Reply:
Ah, that was a tab before, I'll make it one space.